### PR TITLE
RHINENG-14556 remove old code to keep the new service independent

### PR DESCRIPTION
**Pull request by r14chandra**
## Why do we need this change? :thought_balloon:

Eliminating codes that are not necessary for the new backend.

## Documentation update? :memo:

- [ ] Yes
- [X] No

## Security Checklist :lock:

Upon raising this PR please go through [RedHatInsights/secure-coding-checklist](https://github.com/RedHatInsights/secure-coding-checklist)

## :guardsman: Checklist :dart:

- [ ] Bugfix
- [ ] New Feature
- [X] Refactor
- [ ] Unittests Added
- [ ] DRY code
- [ ] Dependency Added
- [ ] DB Migration Added

## Additional :mega:

Feel free to add any other relevant details such as __links, notes, screenshots__, here.

### CODE REVIEW

File: pr_check.sh [hunk 1]
```diff
@@ -25,4 +25,11 @@ source $CICD_ROOT/build.sh
 source $CICD_ROOT/deploy_ephemeral_env.sh
 
 # Run iqe-ros somke tests with ClowdJobInvocation
-source $CICD_ROOT/cji_smoke_test.sh
+# source $CICD_ROOT/cji_smoke_test.sh  # require revisit after the new backed is up
```
Comment (upadhyeammit) [inline pr_check.sh:28]:
```suggestion
# source $CICD_ROOT/cji_smoke_test.sh  # require revisit after the new backend is up
```

**Comment by upadhyeammit** [comment]
@r14chandra I can see we have not touched the models, so with this pr my impression is we go with current models as is and later on tweak it as per requirements?

**Comment by r14chandra** [comment]
> @r14chandra I can see we have not touched the models, so with this pr my impression is we go with current models as is and later on tweak it as per requirements?

Yes, that's true. It’s also possible that what I am removing right now may be needed later.

**Review by upadhyeammit — APPROVED** [review]

**Pull request merged**